Source/WebInspectorUI/UserInterface/Models/CSSProperty.js

```diff
@@ -129,12 +129,20 @@ WI.CSSProperty = class CSSProperty extends WI.Object
         return 0;
     }
 
-    static indexOfCompletionForMostUsedPropertyName(completions)
+    static indexOfCompletionForMostUsedPropertyName(completions, completionPrefix)
     {
+        let exactMatchIndex = -1;
         let highestRankCompletions = completions;
         if (highestRankCompletions.every((completion) => completion instanceof WI.QueryResult)) {
             let highestRankValue = -1;
-            for (let completion of completions) {
+            for (let i = 0; i < completions.length; i++) {
+                let completion = completions[i];
+                // Exact match overrides popularity by usage.
+                if (completionPrefix === WI.CSSCompletions.getCompletionText(completion)){
```

Comment:
since we know that `completion` is a `WI.QueryResult`, we should just use `completion.value` instead

Style: missing space between `)` and `{`